conf/application.conf

```diff
@@ -234,3 +234,24 @@ hbase {
   initialize = false
   initialize = ${?HBASE_INITIALIZE_DB}
 }
+
+kamon {
+  environment {
+    service = "sbr-admin-data"
```

Comment:
For admin data, it may be a good idea to allow the service name to be overriden by an environment variable (using the usual pattern where the default value is followed by another declaration specifying ```service = ${?ENVIRONMENT_VARIABLE_NAME})```.

There will be 3 different instances of the admin data service (paye, vat, ch) - so it might be nice if they can be configured to have a service name of sbr-admin-data-vat for example.

Reply:
Note that to achieve this the Jenkins build will need to change, as each variable needs to be explicitly injected into the deployment (given the way it is currently setup).